README.md

```diff
@@ -1,14 +1,18 @@
-[![](https://img.shields.io/jitpack/v/hyperledger/iroha-java.svg)](https://jitpack.io/#hyperledger/iroha-java/iroha2-dev-SNAPSHOT)
+# Iroha2 Java
 
-# Iroha2-java
+[![](https://img.shields.io/jitpack/v/hyperledger/iroha-java.svg)](https://jitpack.io/#hyperledger/iroha-java/iroha2-dev-SNAPSHOT)
 
-Client library of [Iroha2](https://github.com/hyperledger/iroha2) written in Kotlin
+The repository contains the client library of [Hyperledger Iroha2](https://github.com/hyperledger/iroha2) written in Kotlin. This includes:
 
 - [x] [SDK](./modules/client) to work with Iroha2 API
 - [x] [codegen](./modules/codegen) to generate models into [model](./modules/model) by Iroha2 schema
-- [x] [model](./modules/model) generated models by [codegen](./modules/codegen)
-- [x] [block](./modules/block) genesis classes and some generated datamodels serializers and deserializers
-- [x] [testcontainers](./modules/testcontainers) wrapper for convenient integration testing with Iroha
+- [x] [model](./modules/model) with models generated by [codegen](./modules/codegen)
+- [x] [block](./modules/block) genesis classes and some generated serializers and deserializers for data models
+- [x] [testcontainers](./modules/testcontainers) wrapper for convenient integration testing with Iroha2
+
```

Comment:
![image](https://user-images.githubusercontent.com/17858062/173558203-b54afa82-e3a5-40c2-a3d5-28b5e1e83de1.png)
some more to add I guess

Reply:
added